src/items.cpp

```diff
@@ -1753,89 +1737,86 @@ EquipItemResult equipItem(Item* item, Item** slot, int player)
 		}
 		return EQUIP_ITEM_SUCCESS_NEWITEM;
 	}
-	else
```

Comment:
again the else {} separates scope from the above as preference

Reply:
Fixed in cd73a285